src/RequestsLibrary/RequestsKeywords.py

```diff
@@ -80,7 +80,7 @@ def _create_session(
         s.auth = auth if auth else s.auth
         s.proxies = proxies if proxies else s.proxies
 
-        max_retries = self.builtin.convert_to_integer(max_retries)        
+        max_retries = int(max_retries)        
```

Comment:
With current code the Robot Test Case itself does not fail, just garbage messages are printed to Robot Framework logs.
A really simple test case is:
```
*** Settings ***
Library           RequestsLibrary

*** Test Cases ***
testCreateSession
    Create Session    v_url    http://www.google.com/    max_retries=1
```

It generates the following output.xml log:
```
<kw type="kw" name="RequestsLibrary.Create Session">
	<doc>Create Session: create a HTTP session to a server</doc>
	<arguments>
		<arg>v_url</arg>
		<arg>http://www.google.com/</arg>
		<arg>max_retries=1</arg>
	</arguments>
	<msg timestamp="20160512 11:08:44.226" level="INFO">Creating Session using : alias=v_url, url=http://www.google.com/, headers={},                     cookies=None, auth=None, timeout=None, proxies=None, verify=False,                     debug=0 </msg>
	<msg timestamp="20160512 11:08:44.229" level="INFO">Argument types are:
&lt;type 'unicode'&gt;</msg>
	<status status="PASS" endtime="20160512 11:08:44.229" starttime="20160512 11:08:44.225"/>
</kw>
```

And the debug log shows:
```
20160512 11:08:44.226 - INFO - +------ START KW: RequestsLibrary.Create Session [ v_url | http://www.google.com/ | max_retries=1 ]
20160512 11:08:44.226 - INFO - Creating Session using : alias=v_url, url=http://www.google.com/, headers={},                     cookies=None, auth=None, timeout=None, proxies=None, verify=False,                     debug=0
20160512 11:08:44.226 - DEBUG - Creating session: v_url
**20160512 11:08:44.229 - INFO - Argument types are:
<type 'unicode'>**
20160512 11:08:44.229 - INFO - +------ END KW: RequestsLibrary.Create Session (4)
```

The message is confusing and not required.
It is the expected behavior of `convert_to_integer` keyword of Robot Framework, which is not meant to be used by another keyword.